docusaurus/docs/React/release-guides/emoji-picker.mdx

```diff
@@ -0,0 +1,78 @@
+---
+id: new-emoji-picker-integration
```

Comment:
I would maybe change the name of the file so that it is clear that this is a migration guide for v11